### DISCUSSION TITLE
TypeError: jsPsych.getDisplayElement(...).querySelector(...) is null

### DISCUSSION THREAD

**Discussion by harishbalakrishnan3** [jspsych/jsPsych#598] [Q&A]
I am new to jsPsych. I have created a small experiment. I am able to run it locally without any errors. But, when I host it on the website, I am getting the following errors. Can someone please help?

Website URL: http://harishbalakrishnan.online/f/expa.html

![image](https://user-images.githubusercontent.com/26023040/52929447-7987be80-336a-11e9-8184-43e265044703.png)

**Comment by jodeleeuw** [comment]
It looks like the page at that URL has been removed. Is the problem solved?

**Comment by harishbalakrishnan3** [comment]
Hi! The problem seems to be related to the huge size (~ 1.5GB) of animation frames that I am using as the stimuli for the jspsych-animation plugin. Now, I have reduced the number of animation frames and the size of each frame. I don't see the problem now. 
So, yes we can close this thread. I forgot about this thread. Apologies and thanks!
